tests/meson.build

```diff
@@ -21,10 +21,15 @@ if gtest_dep.found()
     gdbus_gen_header_test_dep = declare_dependency(sources: gdbus_gen_test_src,
       dependencies: gdbus_gen_header_dep)
 
+    nns_ml_agent_common_objs = nns_ml_agent_executable.extract_objects(nns_ml_agent_common_srcs)
+    if (get_option('enable-tizen'))
+      nns_ml_agent_tizen_only_objs = nns_ml_agent_executable.extract_objects(nns_ml_agent_tizen_only_srcs)
+    endif
     executable('machine-learning-agent-test',
-      [nns_ml_agent_srcs, files('daemon/test-dbus-impl.c')],
+      [nns_ml_agent_service_db_srcs, files('daemon/test-dbus-impl.c')],
       dependencies: [ai_service_daemon_deps, gdbus_gen_header_test_dep],
       include_directories: nns_ml_agent_incs,
+      objects: [nns_ml_agent_common_objs, nns_ml_agent_tizen_only_objs],
```

Comment:
`nns_ml_agent_tizen_only_objs` is not defined in non-tizen env. How about add sth like `nns_ml_agent_tizen_only_objs = []`

Reply:
As @jaeyun-jung suggested, I've changed it so that ```nns_ml_agent_tizen_only_objs``` is not needed anymore.